Library/Formula/swagger-codegen.rb

```diff
@@ -0,0 +1,25 @@
+class SwaggerCodegen < Formula
+  desc "Generation of client and server from Swagger definition"
+  homepage "http://swagger.io/swagger-codegen/"
+  url "https://github.com/swagger-api/swagger-codegen/archive/v2.1.4.tar.gz"
+  sha256 "b4965158fb8abbebdfcaab2fdc0c5556001c587a526279d44ea9878f7ef9ef4c"
+  head "https://github.com/swagger-api/swagger-codegen.git"
+
+  depends_on :java => "1.7"
+  depends_on "maven" => :build
+
+  resource "minimal.yaml" do
+    url "https://raw.githubusercontent.com/swagger-api/swagger-editor/9ff93eb21264d1274d9269a3930bfb9903b53b2b/app/spec-files/minimal.yaml"
+    sha256 "9293aaa4968e2ff39d90d2c1be0452234741d7c2649d4b511ab31e2b1ce6ef2d"
+  end
```

Comment:
Since this is really small we could inline it in the test:
```rb
(testpath/"minimal.yaml").write <<-EOS.undent
  ---
  swagger: '2.0'
  info:
    version: 0.0.0
    title: Simple API
  paths:
    /:
      get:
        responses:
          200:
            description: OK
EOS
```